README.md

```diff
@@ -5,23 +5,24 @@
 `z-quantum-core` is a core library of the scientific code for [Orquestra](https://www.zapatacomputing.com/orquestra/) – the platform developed by [Zapata Computing](https://www.zapatacomputing.com) for performing computations on quantum computers.
 
 `z-quantum-core` provides:
+
 - core functionalities required to run other Orquestra modules, such as the `Circuit` class.
 - interfaces for implementing other Orquestra modules, such as backends and optimizers.
 - useful tools to support the development of workflows and other scientific projects; such as time evolution, sampling from probability distribution, etc.
 
-
 ## Usage
 
 ### Workflow
+
 In order to use `z-quantum-core` in your workflow, you need to add it as an `import` in your Orquestra workflow:
 
 ```yaml
 imports:
-- name: z-quantum-core
-  type: git
-  parameters:
-    repository: "git@github.com:zapatacomputing/z-quantum-core.git"
-    branch: "master"
+  - name: z-quantum-core
```

Comment:
was this indentation intended? TBH I like this way of writing `.yaml`s better, but I recall in our docs we usually go with `-` at the container level

Reply:
Just automatic formatting at work here. If we establish prettier as the yaml formatter, this would be the new style.